src/pages/ImportData.js

```diff
@@ -61,6 +62,7 @@ const Importer = class Importer extends Component {
         ['entitiesJson', () => Ajax().Workspaces.workspace(namespace, name).importEntities(url)],
         [Utils.DEFAULT, () => Ajax().Workspaces.workspace(namespace, name).importBagit(url)]
       )
+      notify('success', 'Data imported successfully.')
```

Comment:
It might be nice to put a timeout on this since it's not essential for the user to acknowledge it.